call/BUILD.gn

```diff
@@ -523,7 +523,7 @@ if (rtc_include_tests) {
         "../test/time_controller:time_controller",
         "../video",
         "adaptation:resource_adaptation_test_utilities",
-      "//test/scenario:scenario",
+        "../test/scenario:scenario",
         "//testing/gmock",
```

Comment:
why testing/gmock & test/gtest does not need this while scenario will?

Reply:
`//testing/gmock` is in root directory, while `test/gtest` is a relative path to `third_party/webrtc`.